pipe.go

```diff
@@ -248,13 +252,19 @@ func _newPipe(connFn func() (net.Conn, error), option *ClientOption, r2ps, nobg
 				p.version = int32(vv)
 			}
 		}
+		if az, ok := p.info["availability_zone"]; ok {
+			p.az = az.string
+		}
 		p.onInvalidations = option.OnInvalidations
 	} else {
 		if !option.DisableCache {
 			p.Close()
 			return nil, ErrNoCache
 		}
 		init = init[:0]
+		if !isServerVersionLowerThan6 {
+			init = append(init, []string{"HELLO"})
+		}
```

Comment:
Can we remove `isServerVersionLowerThan6` and just always send `HELLO 2` anyway?

Reply:
[908bc94](https://github.com/redis/rueidis/pull/721/commits/908bc940f1ea3372e4e2236d8426184d2f032163)